### PR TITLE
Use latest ambertools?

### PR DESCRIPTION
**Pull request by hainm**
also a good way to test packing all python build versions into a single AT tar.

### CODE REVIEW

File: devtools/ci/travis/install.sh [hunk 1]
```diff
@@ -35,13 +35,13 @@ else # Otherwise, CPython... go through conda
...
-        python -c "import parmed; print(parmed)"
     else
         # Do not install the full numpy/scipy stack
         conda create -y -n myenv python=$PYTHON_VERSION numpy nose pyflakes=1.0.0 \
             coverage nose-timer python-coveralls
     fi
 
     source activate myenv
+    python -c "import sander; print(sander)"
+    python -c "import parmed; print(parmed)"
```
Comment (swails) [inline devtools/ci/travis/install.sh]: Aren't these going to fail in the test slots that don't install sander?

Comment (hainm) [inline devtools/ci/travis/install.sh]: for my testing. For some reasons sander is not imported properly.

(You might want to turn off notification for this thread :D).

Comment (hainm) [inline devtools/ci/travis/install.sh]: I will reopen later.  ack.

```
The following NEW packages will be INSTALLED:

    ambertools: 17.0-0  http://ambermd.org/downloads/ambertools-rc/conda
    libgcc:     5.2.0-0                                                 

ambertools-17. 100% || Time: 0:00:15   7.96 MB/s
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named sander
<module 'parmed' from 'parmed/__init__.py'>
```

Comment (hainm) [inline devtools/ci/travis/install.sh]: uhm super weird, the ambertools conda works fine for this repo:

https://api.travis-ci.org/jobs/222133069/log.txt?deansi=true (search for ambertools)

```bash

The following NEW packages will be INSTALLED:

    ambertools: 17.0-0 http://ambermd.org/downloads/ambertools-rc/conda

ambertools-17. 100% || Time: 0:00:03  34.51 MB/s

$ tleap -h
-I: Adding /home/travis/miniconda2/dat/leap/prep to search path.
-I: Adding /home/travis/miniconda2/dat/leap/lib to search path.
-I: Adding /home/travis/miniconda2/dat/leap/parm to search path.
-I: Adding /home/travis/miniconda2/dat/leap/cmd to search path.
Usage: /home/travis/miniconda2/bin/teLeap [options]
Options:
 -h         Generate this message.
 -s         Ignore leaprc startup file.
 -I {dir}   Add {dir} to search path.
 -f {file}  Source {file}.


The command "tleap -h" exited with 0.
$ sqm -h
sqm [-O] -i <input> -o <output>
   -O           Overwrite output file if it exists
   -i <input>   Input file
   -o <output>  Output file
   -h           Show this message


The command "sqm -h" exited with 0.
$ cpptraj -h

Usage: cpptraj [-p <Top0>] [-i <Input0>] [-y <trajin>] [-x <trajout>]
               [-c <reference>] [-d <datain>] [-w <dataout>] [-o <output>]
               [-h | --help] [-V | --version] [--defines] [-debug <#>]
               [--interactive] [--log <logfile>] [-tl]
               [-ms <mask>] [-mr <mask>] [--mask <mask>] [--resmask <mask>]
       cpptraj <Top> <Input>
	-p <Top0>        : Load <Top0> as a topology file. May be specified more than once.
	-i <Input0>      : Read input from <Input0>. May be specified more than once.
	-y <trajin>      : Read from trajectory file <trajin>; same as input 'trajin <trajin>'.
	-x <trajout>     : Write trajectory file <trajout>; same as input 'trajout <trajout>'.
	-c <reference>   : Read <reference> as reference coordinates; same as input 'reference <reference>'.
	-d <datain>      : Read data in from file <datain> ('readdata <datain>').
	-w <dataout>     : Write data from <datain> as file <dataout> ('writedata <dataout>).
	-o <output>      : Write CPPTRAJ STDOUT output to file <output>.
	-h | --help      : Print command line help and exit.
	-V | --version   : Print version and exit.
	--defines        : Print compiler defines and exit.
	-debug <#>       : Set global debug level to <#>; same as input 'debug <#>'.
	--interactive    : Force interactive mode.
	--log <logfile>  : Record commands to <logfile> (interactive mode only). Default is 'cpptraj.log'.
	-tl              : Print length of trajectories specified with '-y' to STDOUT.
	-ms <mask>       : Print selected atom numbers to STDOUT.
	-mr <mask>       : Print selected residue numbers to STDOUT.
	--mask <mask>    : Print detailed atom selection to STDOUT.
	--resmask <mask> : Print detailed residue selection to STDOUT.



The command "cpptraj -h" exited with 0.
$ python -c "import parmed; print(parmed)"
<module 'parmed' from '/home/travis/miniconda2/lib/python2.7/site-packages/parmed/__init__.py'>


The command "python -c "import parmed; print(parmed)"" exited with 0.
$ python -c "import sander; print(sander)"
<module 'sander' from '/home/travis/miniconda2/lib/python2.7/site-packages/sander/__init__.py'>


The command "python -c "import sander; print(sander)"" exited with 0.
$ python -c "import pytraj; print(pytraj)"
<module 'pytraj' from '/home/travis/miniconda2/lib/python2.7/site-packages/pytraj/__init__.py'>


The command "python -c "import pytraj; print(pytraj)"" exited with 0.
$ python -c "import nglview; print(nglview)"
<module 'nglview' from '/home/travis/miniconda2/lib/python2.7/site-packages/nglview/__init__.pyc'>


The command "python -c "import nglview; print(nglview)"" exited with 0.

Done. Your build exited with 0.
```

**Comment by hainm** [comment]
shoot, I forget to add `-n myenv`.

**Comment by hainm** [comment]
@swails you might want to activate [Auto Cancellation] ("allows you to only run builds for the latest commits in the queue") on travis settings.

**Comment by hainm** [comment]
finally pass on travis

```
Tests energy action with sander ... ok ([32m1.4023s[0m)
Tests gb energy action with sander ... ok ([32m0.0548s[0m)
Test HMassRepartition on AmberParm ... ok ([32m0.6733s[0m)
Test lmod action on AmberParm ... ok ([32m0.2885s[0m)
Tests the minimize action with pysander and scipy ... RUNNING THE L-BFGS-B CODE
```

File: .travis.yml [hunk 2]
```diff
@@ -20,5 +20,6 @@ matrix:
 
 install:
     - source devtools/ci/travis/install.sh
-    - python setup.py install
+    - pip install .
```
Comment (swails) [inline .travis.yml]: What does this do that `python setup.py install` doesn't?

**Comment by hainm** [comment]
For some reasons, that won't overwrite "trace" of Parmed in AmberTools.

(Switch to pip install does help passing the test. This is my shot in the dark)

Hai

**Comment by hainm** [comment]
https://dubroy.com/blog/so-you-want-to-install-a-python-package/

Pip vs setup.py install

**Comment by hainm** [comment]
smooth. This is my final version.

**Comment by swails** [comment]
Thanks!

**Pull request merged**